apps/wpcom-block-editor/src/common/tracking/wpcom-block-picker-search-term-handler.js

```diff
@@ -1,8 +1,31 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
 /**
  * Internal dependencies
  */
 import tracksRecordEvent from './track-record-event';
 
+const trackSearchTermNotFound = ( event, target ) => {
```

Comment:
Todo: rename this since the function does tracking of the search term also when blocks _are_ found. :-)